### PR TITLE
Remove PXF from the Greenplum repository

### PR DESCRIPTION
**Pull request by frankgh**
The PXF extension has been moved to the PXF repository, and PXF will now
ship as an RPM and DEB, where the extension and the server will ship
together.

### CODE REVIEW

File: GNUmakefile.in [hunk 1]
```diff
@@ -192,7 +192,6 @@ $(call recurse,installcheck-world,contrib/pg_upgrade,check)
 unittest-check:
 	$(MAKE) -C src/backend unittest-check
 	$(MAKE) -C src/bin unittest-check
-	$(MAKE) -C gpcontrib unittest-check
```
Comment (ashwinstar) [inline GNUmakefile.in:195]: do we not have non-pxf unit tests in `gpcontrib`? Seems we should not take out this line.

Comment (frankgh) [reply GNUmakefile.in:195]: we only have unit tests for PXF in gpcontrib, that's why we remove this line. Other extensions do not have unittests. Here's the gpcontrib Makefile change : https://github.com/greenplum-db/gpdb/pull/10798/files#diff-6ff71d1eb879dffb90bfc28bf907d895L104

Comment (ashwinstar) [reply GNUmakefile.in:195]: Interesting. Thanks!

**Review by xinzweb — APPROVED** [review]
w/ @bmdoil, thanks for this PR removing the PXF from the server. For the release process, we need to make the corresponding changes to handle the PXF installation/packaging properly.

**Pull request merged**